tests/NSubstitute.Analyzers.Tests.CSharp/DiagnosticAnalyzerTests/CallInfoAnalyzerTests/ReturnsAsExtensionMethodTests.cs

```diff
@@ -555,10 +505,10 @@ public void Test()
     }}
 }}";
 
-            await VerifyDiagnostic(source);
+            await VerifyNoDiagnostic(source);
```

Comment:
I don't understand why some of these seem to have changed from `VerifyDiagnostic` to `VerifyNoDiagnostic`? Looking at the full file everything seems ok, but from the diff it looks a bit odd?

Reply:
That was my initial change before starting refactoring. Thanks to that I was able to make my life easier when checking if the parser works correctly - as I am throwing an exception when you try to verify diagnostic without specifying them.